tools/c7n_gcp/tests/test_bigquery.py

```diff
@@ -26,6 +26,25 @@ def test_query(self):
             ["gcp:bigquery::cloud-custodian:dataset/devxyz"],
         )
 
+    def test_dataset_delete(self):
+        project_id = 'cloud-custodian'
+        factory = self.replay_flight_data('bq-dataset-delete', project_id=project_id)
```

Comment:
this still needs the recorded test data files to be able to pass. note on gcp we don't currently auto obscure project ids when recording  the way we do in aws/azure.

Reply:
Hi Kapil, thanks for your reviews. I have incorporated it with review comments. Please check and get it merged.